Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppProtectionPolicyiOS/MSFT_IntuneAppProtectionPolicyiOS.psm1

```diff
@@ -1080,8 +1080,8 @@ function Test-TargetResource
 
     $CurrentValues = Get-TargetResource @PSBoundParameters
 
-    Write-Verbose -Message "Current Values: $(Convert-M365DscHashtableToString -Hashtable $CurrentValues)"
-    Write-Verbose -Message "Target Values: $(Convert-M365DscHashtableToString -Hashtable $PSBoundParameters)"
+    Write-Verbose -Message "Current Values: $((Convert-M365DscHashtableToString -Hashtable $CurrentValues) -replace ';', "`r`n")"
```

Comment:
Can you leave this as it was? I like the change, but now it is only implemented for this resource and not for all other resources. I rather change the Convert-M365DscHashtableToString function to do this, so all resources replace the semi colon with an enter.

Please create an issue for this so we can assess the impact and implement separately.